docs/design-documents/20240430-schema-support.md

```diff
@@ -0,0 +1,416 @@
+<!-- markdownlint-disable MD004 MD007 MD033 -->
```

Comment:
Lol that's one way to deal with the linter 😄 I'd prefer if we follow the linter suggestions and change the content to ensure it follows the same style as other documents.

Reply:
Yeah, I don't like it either, but the auto-generated TOC breaks these rules (no HTML, using asterisk, indentation) I should have added a comment explaining  the reason for the above.

Reply:
Can you update the TOC manually when you're done and don't anticipate any more changes? It's not like we'll touch this document again and will need to re-generate it.

Reply:
Sounds like a good idea!